TECHNICAL-SETUP.md

```diff
@@ -0,0 +1,67 @@
+# Technical setup 
+
+
+## Software requirement
+
+To build the project:
+
+- [Golang](https://golang.org/dl/) v1.16 or greater 
+- [`make`](https://www.gnu.org/software/make/) - to use Makefile targets
+- `sha256sum` = to calculate build checksums
+
+To run the [seed scripts](scripts/seeds/README.md)
+
+- OS: linux or macOS
+- [jq](https://stedolan.github.io/jq/) - json processor
+
+For git pre-commit hooks
+
+- python
+- [pre-commit](https://pre-commit.com/#install) - git pre-commit hooks framework 
+- [golangci-lint](https://github.com/golangci/golangci-lint) - linter for Go 
+
+### Swagger 
+
+To generate the swagger UI the following dependencies must be installed
+
+- [`swagger-combine`](https://www.npmjs.com/package/swagger-combine)
+
+
+## Development environment setup (only for contributing)
```

Comment:
do you mean contributing to the codebase?

Reply:
yes, updated